lib/Serialization/SerializedModuleLoader.cpp

```diff
@@ -818,9 +823,14 @@ void swift::serialization::diagnoseSerializedASTLoadFailure(
 bool SerializedModuleLoaderBase::canImportModule(
     std::pair<Identifier, SourceLoc> mID) {
   // Look on disk.
+  SmallVector<char, 0> *moduleInterfacePath = nullptr;
+  std::unique_ptr<llvm::MemoryBuffer> *moduleBuffer = nullptr;
+  std::unique_ptr<llvm::MemoryBuffer> *moduleDocBuffer = nullptr;
+  std::unique_ptr<llvm::MemoryBuffer> *moduleSourceInfoBuffer = nullptr;
```

Comment:
(1) It felt a bit too noisy with all the comments (lots of `/* */`) interleaved with the 7 or 8 arguments, so I put them up earlier.
(2) It leads to poorer readability thanks to uniform syntax highlighting for comments.
(3) In case the arguments get swapped or something, we won't have an implicit conversion this way. Some of them are still the same types, so it doesn't help there 🤷‍♂.

Reply:
Maybe it could be clearer if you named the locals in a way that communicates the intent a little more? `unusedBuffer`, `ignoreDocBuffer`, whatever.